### ISSUE TITLE
Program window is transparent

### ISSUE THREAD

**Issue by thwaller** [kitone/subtitleeditor#9]
Using the current version on Ubuntu 17.10 MATE environment. When I open the program, the window, the portion under the menu bar, is all transparent. But what will then happen, if I move the window, it retains what was under the window before I moved it. I hope that makes sense.

I have attached a screenshot to show an example as it is somewhat difficult for me to explain with confidence.
![selection_003](https://user-images.githubusercontent.com/13359198/34658716-d9a37c14-f3f6-11e7-9f7d-13a4d3673f23.png)

**Comment by dikonov** [comment]
Yes, it does look like a bug and makes new users think that the application is not functional.
It would be nice to paint those areas with background (graph) and black (video).
